variables.scss

```diff
@@ -0,0 +1,5 @@
+$first-color: #131B23;
```

Comment:
I don't like names for this variables, they are totally not self-explanatory. 
Name them as -> `main-green`, `light-green`. `bg-color`, etc. Something saying about this color ;)